.github/workflows/codeql-analysis.yml

```diff
@@ -22,15 +22,16 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v2.3.4
 
-    - name: Setup dotnet (for SQL generation)
+    - name: Setup dotnet from global.json
```

Comment:
I thought you removed `global.json`?

Reply:
There were previously two. We keep a global JSON at the root to specify the .NET SDK used to build the solution. But I removed the one I previously added that was in the *Microsoft.Health.Dicom.Functions* folder. I initially imaged we'd use a separate SDK for building the functions, but instead we'll just keep the .NET 5.0 SDK for the solution overall.